src/app/components/set-pin.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="modal fade" tabindex="-1" role="dialog" aria-modal="true" aria-labelledby="confirmUserTitle">
```

Comment:
The `aria-labelledby` seems incorrect. I don't believe we have any title for this modal.